sdk/src/gateway/client.ts

```diff
@@ -72,6 +73,7 @@ type GatewayCreateOrderResponse = {
 type GatewayStartOrderResult = GatewayCreateOrderResponse & {
     bitcoinAddress: string,
     satoshis: number;
+    psbtBase64: string;
```

Comment:
If we already want this as bi-directional, we could name this `unsignedTx` or something like that so it would be a valid name for both a BTC as well as an EVM tx. No strong preference right now though. Maybe also cleaner to have `psbtBase64?` and eventually add `rawEvmTx?`or something like that.

Reply:
Yeah I think cleaner to keep separate